src/test/lit-element_test.ts

```diff
@@ -727,7 +762,7 @@ suite('LitElement', () => {
         el.info.push('updateComplete');
         assert.deepEqual(
             el.info,
-            [ 'shouldUpdate', 'before-update', 'render', 'finishUpdate', 'updateComplete' ]);
+            [ 'shouldUpdate', 'before-update', 'render', 'finishUpdate', 'firistUpdated', 'updateComplete' ]);
```

Comment:
Spelling mistake... "firist" should be "first"

Reply:
Done.